.github/workflows/notify_codeowners.py

```diff
@@ -0,0 +1,173 @@
+import github
+import click
+import urllib.request
+from pathlib import Path
+from typing import List, Tuple
+import re
+import fnmatch
+import glob
+import os
+import json
+
+# Github already take care
+# of notifying users with write access
+BLACKLIST = [
+    "tensorflow/sig-addons-maintainers",
+    "facaiy",
+    "seanpmorgan",
+    "squadrick",
+    "shun-lin",
+    "windqaq",
+    "qlzh727",
+    "guillaumekln",
+    "helinwang",
```

Comment:
Doesn't have write access. Updated the confusing codeowners in another repo

Reply:
Done. Please merge this PR before updating the codeowners. A check runs in the CI to verify that CODEOWNERS is valid.